src/cascadia/LocalTests_SettingsModel/DeserializationTests.cpp

```diff
@@ -1075,141 +657,19 @@ namespace SettingsModelLocalTests
             ]
         })" };
 
-        const auto settings0Json = VerifyParseSucceeded(settings0String);
-
-        auto settings = winrt::make_self<implementation::CascadiaSettings>();
-        settings->_ParseJsonString(DefaultJson, true);
-        settings->LayerJson(settings->_defaultSettings);
-        VERIFY_ARE_EQUAL(2u, settings->_allProfiles.Size());
-        VERIFY_IS_TRUE(settings->_allProfiles.GetAt(0).HasGuid());
-        VERIFY_IS_TRUE(settings->_allProfiles.GetAt(1).HasGuid());
-        VERIFY_ARE_EQUAL(L"Windows PowerShell", settings->_allProfiles.GetAt(0).Name());
-        VERIFY_ARE_EQUAL(L"Command Prompt", settings->_allProfiles.GetAt(1).Name());
-
-        Log::Comment(NoThrowString().Format(
-            L"Parse the user settings"));
-        settings->_ParseJsonString(settings0String, false);
-        settings->LayerJson(settings->_userSettings);
-
-        VERIFY_ARE_EQUAL(5u, settings->_allProfiles.Size());
-        VERIFY_IS_TRUE(settings->_allProfiles.GetAt(0).HasGuid());
-        VERIFY_IS_TRUE(settings->_allProfiles.GetAt(1).HasGuid());
-        VERIFY_IS_TRUE(settings->_allProfiles.GetAt(2).HasGuid());
-        VERIFY_IS_FALSE(settings->_allProfiles.GetAt(3).HasGuid());
-        VERIFY_IS_FALSE(settings->_allProfiles.GetAt(4).HasGuid());
-        VERIFY_ARE_EQUAL(L"Windows PowerShell", settings->_allProfiles.GetAt(0).Name());
-        VERIFY_ARE_EQUAL(L"Command Prompt", settings->_allProfiles.GetAt(1).Name());
-        VERIFY_ARE_EQUAL(L"ThisProfileIsGood", settings->_allProfiles.GetAt(2).Name());
-        VERIFY_ARE_EQUAL(L"ThisProfileShouldNotLayer", settings->_allProfiles.GetAt(3).Name());
-        VERIFY_ARE_EQUAL(L"NeitherShouldThisOne", settings->_allProfiles.GetAt(4).Name());
-    }
-
-    void DeserializationTests::TestExplodingNameOnlyProfiles()

```

Comment:
How do we validate that this invariant is still true?

Reply:
I considered this test in particular a duplicate of `TestLayeringNameOnlyProfiles` which is extremely similar. So I only fixed one of the two tests. Did I maybe miss something?